charts/refinery/values.yaml

```diff
@@ -496,6 +496,9 @@ podLabels: {}
 
 podAnnotations: {}
 
+deploymentAnnotations: {}
+deploymentRedisAnnotations: {}
```

Comment:
Lets move this under the `redis` configuration section and call it `annotations`.